react/react1/week1/src/components/Note.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { tasks } from "../todo-list";
+
+function NoteList(props) {
+  return (
+    <ul>
+      <li>{props.todo}</li>
+      <li>{props.time}</li>
+    </ul>
+  );
+}
+function Note() {
+  return tasks.map((e) => {
+    return <NoteList todo={e.todo} time={e.time} />;
```

Comment:
You should have a "key" property, so that it renders correctly 
Please read the docs for more info https://react.dev/learn/rendering-lists